### PR TITLE
Examples config

### PR DESCRIPTION
**Pull request by robert-zaremba**

### CODE REVIEW

**Comment by robert-zaremba** [comment]
Note: I've updated the port in default network to the default algorand node port (8080).

File: examples/algob.config-shared.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    name: "bob-account",
+    addr: "2ILRL5YU3FZ4JDQZQVXEZUYKEWF7IEIGRRCPCMI36VKSGDMAS6FHSBXZDQ",
+    mnemonic: "caution fuel omit buzz six unique method kiwi twist afraid monitor song leader mask bachelor siege what shiver fringe else mass hero deposit absorb tooth"
+  }, {
+    name: "alice-account",
+    addr: "M7VR2MGHI35EG2NMYOF3X337636PIOFVSP2HNIFUKAG7WW6BDWDCA3E2DA",
+    mnemonic: "quick stage planet wild coffee whale build brisk forest leave same segment armed alter fog know run distance excess galaxy limb talent nut able noodle"
+  }, {
+    name: "voting-admin-account",
+    addr: "WHVQXVVCQAD7WX3HHFKNVUL3MOANX3BYXXMEEJEJWOZNRXJNTN7LTNPSTY",
```
Comment (ratik21) [inline examples/algob.config-shared.js:49]: This is the same addr as `elon-musk-account`. Is that ok?

File: examples/algob.config-shared.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+]);
+
+
+let defaultCfg = {
+  host: "http://localhost",
+  // if you are running a local node, you can find port and token values for your algod in
+  // $ALGORAND_DATA/algod.net and $ALGORAND_DATA/algod.token files.
+  port: 8080,
+  token: "aade468d25a7aa48fec8082d6a847c48492066a2741f3731e613fdde086cd6e9",
+
```
Comment (ratik21) [inline examples/algob.config-shared.js:61]: how about adding a comment for `kmdcfg` here? `// to load accounts from kmd check /sample-project...`

File: examples/algob.config-shared.js (hunk 1 above)
Comment (amityadav0) [reply examples/algob.config-shared.js:49]: yes, I don't think that will be problem unless we don't use them in same examples.
But we can change for more clarity

Comment (amityadav0) [inline examples/algob.config-shared.js:49]:
```suggestion
    addr: "L4VD3C7THL6UNR62BDSDNPBCIBTPJ3PQPLWKGYNG4Y34RB6ITO4TNNHZ6Y",
```

File: examples/algob.config-shared.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    addr: "2ILRL5YU3FZ4JDQZQVXEZUYKEWF7IEIGRRCPCMI36VKSGDMAS6FHSBXZDQ",
+    mnemonic: "caution fuel omit buzz six unique method kiwi twist afraid monitor song leader mask bachelor siege what shiver fringe else mass hero deposit absorb tooth"
+  }, {
+    name: "alice-account",
+    addr: "M7VR2MGHI35EG2NMYOF3X337636PIOFVSP2HNIFUKAG7WW6BDWDCA3E2DA",
+    mnemonic: "quick stage planet wild coffee whale build brisk forest leave same segment armed alter fog know run distance excess galaxy limb talent nut able noodle"
+  }, {
+    name: "voting-admin-account",
+    addr: "WHVQXVVCQAD7WX3HHFKNVUL3MOANX3BYXXMEEJEJWOZNRXJNTN7LTNPSTY",
+    mnemonic: "resist derive table space jealous person pink ankle hint venture manual spawn move harbor flip cigar copy throw swap night series hybrid chest absent art"
```
Comment (amityadav0) [inline examples/algob.config-shared.js:50]:
```suggestion
    mnemonic: "atalog rug strike plug scorpion memory crash act work actor affair tragic fossil ahead book toward trip broccoli wide repair clerk early fog abstract kingdom"
```

File: examples/algob.config-shared.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    name: "john-account",
+    addr: "2UBZKFR6RCZL7R24ZG327VKPTPJUPFM6WTG7PJG2ZJLU234F5RGXFLTAKA",
+    mnemonic: "found empower message suit siege arrive dad reform museum cake evoke broom comfort fluid flower wheat gasp baby auction tuna sick case camera about flip"
+  }, {
+    name: "bob-account",
+    addr: "2ILRL5YU3FZ4JDQZQVXEZUYKEWF7IEIGRRCPCMI36VKSGDMAS6FHSBXZDQ",
+    mnemonic: "caution fuel omit buzz six unique method kiwi twist afraid monitor song leader mask bachelor siege what shiver fringe else mass hero deposit absorb tooth"
+  }, {
+    name: "alice-account",
+    addr: "M7VR2MGHI35EG2NMYOF3X337636PIOFVSP2HNIFUKAG7WW6BDWDCA3E2DA",
```
Comment (amityadav0) [inline examples/algob.config-shared.js:45]:
```suggestion
    addr: "EDXG4GGBEHFLNX6A7FGT3F6Z3TQGIU6WVVJNOXGYLVNTLWDOCEJJ35LWJY",
```

File: examples/algob.config-shared.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+    mnemonic: "quick stage planet wild coffee whale build brisk forest leave same segment armed alter fog know run distance excess galaxy limb talent nut able noodle"
```
Comment (amityadav0) [inline examples/algob.config-shared.js:46]:
```suggestion
    mnemonic: "brand globe reason guess allow wear roof leisure season coin own pen duck worth virus silk jazz pitch behave jazz leisure pave unveil absorb kick"
```

File: examples/algob.config-shared.js (hunk 1 above)
Comment (robert-zaremba) [reply examples/algob.config-shared.js:49]: I will  remove `voting-admin` at all.

File: examples/algob.config-shared.js (hunk 3 above)
Comment (robert-zaremba) [reply examples/algob.config-shared.js:50]: I will remove voting-admin and reuse other account

File: examples/algob.config-shared.js (hunk 2 above)
Comment (robert-zaremba) [reply examples/algob.config-shared.js]: added.

**Review by ratik21 — APPROVED** [review]
Thanks for the clean up.

**Comment by ratik21** [comment]
@amityadav0 could you confirm:
image
This transaction is failing (which should have passed): [https://github.com/scale-it/algorand-builder/blob/master/examples/asa/scripts/transfer/gold-sc.js#L30]()

**Comment by ratik21** [comment]
Ah, nvm. The smart contract needs to be updated.

**Comment by ratik21** [comment]
Please check: `yarn run algob clean` is not clearing `cache/artifacts`. This is because the root path has changed to `/examples` only (rather than `/examples/<project>`).
image

File: examples/asa/scripts/transfer/gold-sc.js [hunk 6]
```diff
@@ -35,7 +35,7 @@ async function run(runtimeEnv, deployer) {
 
   // Transaction FAIL - sender should be the delegator i.e account which signed the lsig (goldOwner in this case)
   txnParam.amount = 100;
-  txnParam.toAccountAddr = bobAccount.addr;
+  txnParam.toAccountAddr = bob.addr;
```
Comment (ratik21) [inline examples/asa/scripts/transfer/gold-sc.js:38]:
```suggestion
    txnParam.fromAccount = bob;
```

Comment (robert-zaremba) [reply examples/asa/scripts/transfer/gold-sc.js]: Thanks for spotting!

**Pull request merged**